### ISSUE TITLE
CanZE Beta expired message after updating to iOS 16

### ISSUE THREAD

**Issue by jrnormanuk** [fesch/CanZE#756]
Hi
After trying several dongles I was getting good results with the iOS CanZE app. I just updated to iOS 16 and I’m getting a message ‘beta expired’ when I try to launch CanZE. When I follow the link on the homepage to TestFlight to reinstall I get a message saying the program is not accepting new beta testers now. 
Is the iOS upgrade a coincidence or have I locked myself out somehow?
Best, John

**Comment by ik2wxx** [comment]
Sorry I forgot to enable TestFlight for new buildShould be be ok nowRobertoIl giorno 23 set 2022, alle ore 17:00, jrnormanuk ***@***.***> ha scritto:﻿
Hi
After trying several dongles I was getting good results with the iOS CanZE app. I just updated to iOS 16 and I’m getting a message ‘beta expired’ when I try to launch CanZE. When I follow the link on the homepage to TestFlight to reinstall I get a message saying the program is not accepting new beta testers now.
Is the iOS upgrade a coincidence or have I locked myself out somehow?
Best, John
